### DISCUSSION TITLE
Sidebar won't remember size when item opened

### DISCUSSION THREAD

**Discussion by jb2025b** [Floorp-Projects/Floorp#871] [Feature Request]
It always defaults to a narrow size and I have to drag the borders. I don't have this problem with Edge, and it's one thing that is keeping me from jumping to Floorp. 

Youtube music works fine and remembers. Messenger and Google Messages defaults to small width.

**Comment by CutterKnife** [comment]
The default width of the web panel can be changed. `about:preferences#bSB`
However, the BMS currently has several bugs. They will be fixed in the near future.

**Comment by TheChilledBuffalo** [comment]
This issue has been fixed and will be shipped with the next update.

Make sure to use the second last button, beside the close button, to save your width preferences for each sidebar app.

![image](https://github.com/Floorp-Projects/Floorp/assets/87632612/27f5ff38-372b-4ce1-8744-99ce199b8332)

**Comment by g-gundam** [comment]
## Panel Width Regression

I just tried Floorp 12 for the first time, and this feature is no longer present.

<img width="482" height="84" alt="image" src="https://github.com/user-attachments/assets/08f4ee8b-f9fc-40e9-9294-8cc369fc9140" />

- As you can see, the icon for remembering the width of a panel is no longer present.
- **Will it come back?**

**Reply by CutterKnife** [reply]
You can change the default width of the panels in Floorp Hub > Panel Sidebar.

**Reply by g-gundam** [reply]
Thanks for the hint.  I didn't know this existed.  about:hub#/features/sidebar

<img width="1138" height="1509" alt="image" src="https://github.com/user-attachments/assets/001f05b7-4364-45ea-8873-56032b0e6971" />
